modules/publication/templates/publication_submission_confirmation.tpl

```diff
@@ -0,0 +1,9 @@
+Subject: [LORIS] Project proposal submission confirmation
+
+Hello,
+This is to confirm your project proposal "{$Title}" received on "{$Date}".
+We will notify you once your proposal has been reviewed. In the mean time, you
+can edit your project proposal through the following link: "{$URL}".
+
+Thank you,
+The LORIS Team
```

Comment:
It's the $(project name) team that the email is sent on behalf of, not the LORIS team.